app/views/contexts/section_summary_context.py

```diff
@@ -25,7 +25,7 @@ def __init__(
         metadata: Mapping,
         routing_path: RoutingPath,
         current_location: Location,
-        return_to: str = "section-summary",
+        return_to: Optional[str] = "section-summary",
```

Comment:
`section-summary` should be passed when `SectionSummaryContext` is called in a section summary (i.e. not final) context rather than defaulted here.